pkg/controller/controller_test.go

```diff
@@ -2456,11 +2459,47 @@ func assertNumEvents(t *testing.T, strings []string, number int) {
 	}
 }
 
+// failfFunc is a type that defines the common signatures of T.Fatalf and
+// T.Errorf.
+type failfFunc func(t *testing.T, msg string, args ...interface{})
+
+func fatalf(t *testing.T, msg string, args ...interface{}) {
+	t.Log(string(debug.Stack()))
+	t.Fatalf(msg, args...)
```

Comment:
Yes. I will open a follow-up issue to try and fix this throughout. (This isn't the only place where this is happening.)